src/components/editMeasure/editor/MeasureEditor.tsx

```diff
@@ -355,17 +361,61 @@ const MeasureEditor = () => {
               {valuesetMsg}
             </SuccessText>
           )}
-          {!processing && (
-            <MadieEditor
-              onChange={(val: string) => handleMadieEditorValue(val)}
-              value={editorVal}
-              inboundAnnotations={elmAnnotations}
-              inboundErrorMarkers={errorMarkers}
-              height="calc(100vh - 135px)"
-              readOnly={!canEdit}
-              setOutboundAnnotations={setOutboundAnnotations}
-            />
-          )}
+          {!processing &&
+            (featureFlags?.qdmCodeSearch && isQDM ? (
+              <MadieTerminologyEditor
+                onChange={(val: string) => handleMadieEditorValue(val)}
+                value={editorVal}
+                inboundAnnotations={elmAnnotations}
+                inboundErrorMarkers={errorMarkers}
+                height="calc(100vh - 135px)"
+                readOnly={!canEdit}
+                setOutboundAnnotations={setOutboundAnnotations}
+              />
+            ) : (
+              <>
+                <MadieEditor
+                  onChange={(val: string) => handleMadieEditorValue(val)}
+                  value={editorVal}
+                  inboundAnnotations={elmAnnotations}
+                  inboundErrorMarkers={errorMarkers}
+                  height="calc(100vh - 135px)"
+                  readOnly={!canEdit}
+                  setOutboundAnnotations={setOutboundAnnotations}
+                />
+                <div
```

Comment:
the stylings for the footer section is different when compared to the ones which we have when no right and left panel are involved, so I had to used the buttons section twice. when we implement the same functionality for QI Core at that time it would be just easy to remove that part of code without looking out for any conditions

Reply:
that's weird we have two different styles

Reply:
I confirmed with Michael about discrepencies in current editor and figma styles and I have changed the current styles to match the figma styles